backend/YouTubeV2/Program.cs

```diff
@@ -22,8 +24,38 @@
 builder.Services.AddControllers();
 // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
 builder.Services.AddEndpointsApiExplorer();
-builder.Services.AddSwaggerGen();
 
+builder.Services.AddSwaggerGen(c =>
```

Comment:
the whole specifing token in swagger work weird rn. I would like it to work like logging into out app - so i specify email and password and it should add token to my requests. also endpoints that require authentication probably shouldnt be awailable at all before we login in swagger